src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityCreateConnectionRequestController.php

```diff
@@ -0,0 +1,141 @@
+     * @throws \Exception
+     */
+    public function connectionRequestFromEntityAction(
+        Request $request,
+        int $serviceId,
+        string $manageId,
+        string $environment
+    ): Response {
+        $this->validateServiceIsAllowed($serviceId, $manageId, $environment);
+
+        $manageEntity = $this->entityService->getManageEntityById($manageId, $environment);
+        $applicant = $this->authorizationService->getContact();
+        $command = new CreateConnectionRequestCommand($manageEntity, $applicant);
+
+        $form = $this->createForm(ConnectionRequestContainerType::class, $command);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            $entity = $this->entityService->getManageEntityById($manageId, $environment);
+            $parameters = ['entityName' => $entity->getMetaData()->getNameEn()];
+
+            if ($this->isCancelAction($form) || !count($command->getConnectionRequests())) {
+                return $this->render('@Dashboard/EntityPublished/publishedProduction.html.twig', $parameters);
+            }
+
+            if ($form->isValid()) {
+                $this->commandBus->handle($command);
+                return $this->render(
+                    '@Dashboard/EntityPublished/publishedProductionAndConnectionRequest.html.twig',
+                    $parameters
+                );
+            }
+        }
+
+        return $this->render(
+            '@Dashboard/EntityPublished/createConnectionRequest.html.twig',
+            ['form' => $form->createView()]
+        );
+    }
+
+    /**
+     * @Method({"GET", "POST"})
+     * @Route("/entity/create-connection-request-from-overview/{environment}/{manageId}/{serviceId}", name="entity_published_create_connection_request_from_overview")
+     * @throws \Exception
+     */
+    public function connectionRequestFromOverviewAction(
+        Request $request,
+        int $serviceId,
+        string $manageId,
+        string $environment
+    ): Response {
+        $this->validateServiceIsAllowed($serviceId, $manageId, $environment);
+
+        $manageEntity = $this->entityService->getManageEntityById($manageId, $environment);
+        $applicant = $this->authorizationService->getContact();
+        $command = new CreateConnectionRequestCommand($manageEntity, $applicant);
+
+        $form = $this->createForm(ConnectionRequestContainerFromOverviewType::class, $command);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            if ($this->isCancelAction($form) || !count($command->getConnectionRequests())) {
+                return $this->returnToOverview($serviceId);
+            }
+
+            if ($form->isValid()) {
+                $this->commandBus->handle($command);
+                return $this->render(
+                    '@Dashboard/EntityPublished/publishedConnectionRequest.html.twig'
+                );
```

Comment:
This should be a redirect response to a stand alone 'thank you' endpoint (as per point 2 of the functional feedback :point_up_2: ). 

Also semantically the name of the twig template is not very good. The request is not published. It stores a Jira ticket for the Service Desk to action at some point.